my-code/src/components/MovieRating/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Container } from './styles';
+import logoImdb from '../../assets/logo-imdb.svg';
+import logoRotten from '../../assets/logo-rotten-tomatoes.svg';
+
+export default function MovieRating({ rating }) {
+  let logo;
```

Comment:
There is nothing wrong here, but it could be a little bit more elegant, some ideas:
```js
  let logo = logoRotten;
  let logoCssClass = 'logo-rotten';

  if (rating.Source === 'Internet Movie Database') {
    logo = logoImdb;
    logoCssClass = 'logo-imdb';
  }
```

Or:

```js
  const isImdb = rating.Source === 'Internet Movie Database'
  
  const logo = isImdb ? logoImdb : logoRotten;
  const logoCssClass = isImdb ? 'logo-imdb' : 'logo-rotten';
```

Alright? What do you think?

Reply:
I'll stick with the second option you provided. It uses `const` and fewer lines.